gitless/cli/gl_switch.py

```diff
@@ -30,6 +30,7 @@ def main(args, repo):
   if not b:
     pprint.err('Branch {0} doesn\'t exist'.format(args.branch))
     pprint.err_exp('to list existing branches do gl branch')
+    pprint.err_exp('to create a new branch do gl branch -c feature/foo')
```

Comment:
You could pass in the name of the branch the user provided:

`pprint.err_exp('to create a new branch do gl branch -c {0}'.format(args.branch))`